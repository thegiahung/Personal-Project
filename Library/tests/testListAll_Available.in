list all
list all long
list available
list available long
exit
