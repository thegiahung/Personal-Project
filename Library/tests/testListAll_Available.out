user: No books in system.

user: No books in system.

user: No books in system.

user: No books in system.

user: Ending Library process.
